ee/desktop/server/server.go

```diff
@@ -36,6 +36,7 @@ func New(logger log.Logger, authToken string, socketPath string, shutdownChan ch
 
 	authedMux := http.NewServeMux()
 	authedMux.HandleFunc("/shutdown", desktopServer.shutdownHandler)
+	authedMux.HandleFunc("/ping", pongHandler)
```

Comment:
I'd have called this `pingHandler` -- while it might return pongs, it's handling pings.